scripts/ci/code_scanning/Dockerfile-csi-controller-dep-code-scan

```diff
@@ -0,0 +1,25 @@
+# Copyright IBM Corporation 2019.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.6.9
+
+RUN pip3 install safety
+
+COPY controller ./controller
+RUN mkdir /results
+
+WORKDIR .
+
+ENTRYPOINT ["safety", "check", "-o", "/results/code_scan_controller_dep", "-r", "/controller/requirements.txt"]
```

Comment:
consider ending all results file names with "_res", or remove all "_res" endings, for uniformity